tests/ui/perfherder/graphs-view/graphs_view_test.jsx

```diff
@@ -35,6 +36,7 @@ const graphData = createGraphData(
   [],
   [...graphColors],
   [...graphSymbols],
+  [...otherAlerts],
```

Comment:
Since this alerts having something in common with each other, maybe a better name for this variable will be commonAlerts?